Firestore/Source/Auth/FSTCredentialsProvider.h

```diff
@@ -38,11 +39,11 @@ NS_ASSUME_NONNULL_BEGIN
 @interface FSTGetTokenResult : NSObject
 
 - (instancetype)init NS_UNAVAILABLE;
-- (instancetype)initWithUser:(FSTUser *)user
+- (instancetype)initWithUser:(const firebase::firestore::auth::User &)user
                        token:(NSString *_Nullable)token NS_DESIGNATED_INITIALIZER;
 
 /** The user with which the token is associated (used for persisting user state on disk, etc.). */
-@property(nonatomic, nonnull, readonly) FSTUser *user;
+@property(nonatomic, nonnull, readonly) firebase::firestore::auth::User *user;
```

Comment:
mark this property `assign`. As above can be a const reference.

Reply:
done